packages/react-components/react-motion/library/tsconfig.lib.json

```diff
@@ -2,7 +2,7 @@
   "extends": "./tsconfig.json",
   "compilerOptions": {
     "noEmit": false,
-    "lib": ["ES2019", "dom"],
+    "lib": ["ES2019", "dom", "ES2020.Promise"],
```

Comment:
Fluent does not compile to ES2020 but this `allSettled` is supported by our support matrix 

![image](https://github.com/microsoft/fluentui/assets/20744592/ae67d411-63b8-4b45-a01b-1aed330dd160)